### PR TITLE
feat: split up into separate packages, load CSS as JS

### PR DESCRIPTION
**Pull request by jorenbroekema**

### CODE REVIEW

File: mdjs-layout/src/MdjsLayout.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 import { html, LitElement } from 'lit';
 import { unsafeHTML } from 'lit/directives/unsafe-html.js';
-import '../mdjs-layout-controls/mdjs-layout-controls.js';
-import './mdjs-styles.css';
+import '../../mdjs-layout-controls/mdjs-layout-controls.js';
+import styles from './mdjs-styles.css.js';
+
+const sheet = new CSSStyleSheet();
```
Comment (gqio) [inline mdjs-layout/src/MdjsLayout.js:6]: You need a polyfill for being able to use this I think

File: mdjs-layout/src/mdjs-styles.css.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
+export default `
```
Comment (gqio) [inline mdjs-layout/src/mdjs-styles.css.js:1]: please use `/* css */`blabllaba`` then you have the highlighting 😎

File: mdjs-layout/src/MdjsLayout.js (hunk 1 above)
Comment (jorenbroekema) [reply mdjs-layout/src/MdjsLayout.js:7]: Added polyfill

File: mdjs-layout/src/mdjs-styles.css.js (hunk 2 above)
Comment (jorenbroekema) [reply mdjs-layout/src/mdjs-styles.css.js]: ah nice, done

File: mdjs-layout/src/MdjsLayout.js (hunk 1 above)
Comment (jorenbroekema) [reply mdjs-layout/src/MdjsLayout.js:7]: I checked also on firefox, works nicely there :)

**Review by gqio — APPROVED** [review]

**Pull request merged**